render-wasm/src/render.rs

```diff
@@ -273,27 +271,40 @@ impl RenderState {
         &mut self,
         shape: &mut Shape,
         modifiers: Option<&Matrix>,
-        clip_bounds: Option<skia::Rect>,
+        clip_bounds: Option<(Rect, Matrix)>,
     ) {
-        if let Some(modifiers) = modifiers {
-            self.drawing_surface.canvas().concat(&modifiers);
-        }
+        if let Some((bounds, transform)) = clip_bounds {
+            self.drawing_surface.canvas().concat(&transform);
+            self.drawing_surface
+                .canvas()
+                .clip_rect(bounds, skia::ClipOp::Intersect, true);
 
-        let center = shape.bounds().center();
+            // Uncomment this to debug the bounds for the clip
+            /*
```

Comment:
Could we enable this just in debug mode instead of leaving it commented?

Reply:
ok